src/lib-sana/Graph/Graph.cpp

```diff
@@ -1,21 +1,37 @@
 #include "Graph.hpp"
+#include "Utils.hpp"
 #include <vector>
 #include <fstream>
 #include <string>
 #include <stdexcept>
 #include <unordered_set>
 #include <utility>
 #include <sstream>
-#include "Utils.hpp"
 
 using namespace std;
 
+string Graph::getName() const {
+    return Graph::name;

```

Comment:
Also, I don't think this would compile cause name is not static member. Should be just `return name;`;